sdk/cosmosdb/cosmos/test/public/functional/container.spec.ts

```diff
@@ -485,3 +485,28 @@ describe("container.create", function () {
     assertThrowsAsync(() => database.containers.create(containerRequest));
   });
 });
+
+describe("container.deleteAllItemsForPartitionKey", function () {
+  it("should delete all items for partition key", async function () {
```

Comment:
_for a given partition key value_

Reply:
Please add a test case for multi partition container as well.